scenes/game/local_game.gd

```diff
@@ -1636,6 +1636,65 @@ class Player:
 			if old_pos == buddy_old_pos:
 				handle_on_buddy_boosts(false)
 
+	func move_in_direction_by_amount(go_left : bool, amount : int, stop_at_opponent : bool, movement_type : String):
+		var events = []
+		var direction = -1 if go_left else 1
```

Comment:
personally i would have just passed in direction as a 1 or -1 int, but this is less open to passing in weirdo stuff that is not 1 or -1.